pipelines/_DiffimDRP.yaml

```diff
@@ -1,21 +1,50 @@
 description: Image differencing branch of the DRP pipeline for generating lightcurves
 tasks:
-  selectVisits:
+  selectGoodSeeingVisits:
     class: lsst.pipe.tasks.selectImages.BestSeeingQuantileSelectVisitsTask
     config:
-      connections.goodVisits: bestSeeingVisitsDict
+      connections.goodVisits: goodSeeingVisits
   templateGen:
     class: lsst.pipe.tasks.assembleCoadd.CompareWarpAssembleCoaddTask
     config:
         doSelectVisits: true
         assembleStaticSkyModel.doSelectVisits: true
-        connections.selectedVisits: bestSeeingVisitsDict
-        connections.outputCoaddName: bestSeeing
+        connections.selectedVisits: goodSeeingVisits
+        connections.outputCoaddName: goodSeeing
+        # outputCoaddName isn't filling template for contract check
+        connections.coaddExposure: goodSeeingCoadd
+  imageDifference:
+    class: lsst.pipe.tasks.imageDifference.ImageDifferenceTask
+    config:
+        doDecorrelation: True
+        detection.thresholdValue: 5.0
+        doSelectSources: False
+        doScaleDiffimVariance: True
+        coaddName: goodSeeing
+        getTemplate.coaddName: goodSeeing
+        getTemplate.warpType: direct
+        connections.coaddName: goodSeeing
+        # outputCoaddName isn't filling template for contract check
```

Comment:
I don't understand this comment.

Reply:
Yeah..... I'll file a ticket and put a TO DO:
`coaddExposures="{fakesType}{coaddName}Coadd{warpTypeSuffix}"` and the defaults are:
```
                                     defaultTemplates={"coaddName": "deep",
                                                       "skyMapName": "deep",
                                                       "warpTypeSuffix": "",
                                                       "fakesType": ""}
```
If I specify the template as `coaddName: goodSeeing`, I'd expect that `coaddExposures` would resolve to `goodSeeingCoadd`. And I believe it does, but the contracts checker doesn't recognize it. So I'm over-specifying the connections here, so make the contracts checker happy.